test/mysql/Dockerfile

```diff
@@ -1,23 +1,27 @@
+FROM debian:bookworm-slim
+
 ARG MYSQL_VERSION
 
-FROM mysql:${MYSQL_VERSION}-debian
 # Make all apt-get commands non-interactive. Setting this as an ARG will apply to the entire
 # build phase, but not leak into the final image and run phase.
 ARG DEBIAN_FRONTEND=noninteractive
 
-# MySQL 5.7 has been EOL'd and is no longer being built by the official mysql docker maintainers:
-# https://github.com/docker-library/mysql/pull/1019
-# Copy the gpg key from the latest MySQL image.
-COPY --from=mysql:8.0-debian /etc/apt/keyrings/mysql.gpg /etc/apt/keyrings/mysql.gpg
-
-# Install the MySQL test package so that we can get test plugins.
-# https://github.com/docker-library/mysql/issues/1040
+# Install MySQL from apt repositories.
 RUN set -eux \
-    # Install MySQL test package.
     && apt-get update \
-    && apt-get install --yes mysql-community-test
+    && apt-get install -y --no-install-recommends \
+        ca-certificates \
+        wget \
+        gnupg \
+        lsb-release \
+    && echo "deb https://repo.mysql.com/apt/debian/ $(lsb_release -sc) mysql-${MYSQL_VERSION}" > /etc/apt/sources.list.d/mysql.list \
+    && apt-key adv --keyserver pgp.mit.edu --recv-keys A8D3785C 3A79BD29 \
```

Comment:
I followed the instructions in https://dev.mysql.com/doc/refman/8.4/en/linux-installation-apt-repo.html#repo-qg-apt-repo-manual-setup, but `#15 76.64 gpg: keyserver receive failed: No keyserver available`.

I haven't missed dealing with GPG. 😓